src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -267,7 +262,7 @@ internal static bool AllowDetachedSignature()
                 return s_allowDetachedSignature.Value;
             }
 
-            long numericValue = GetNetFxSecurityRegistryValue("SignedXmlAllowDetachedSignature", 0);
+            long numericValue = 0;
             bool allowDetachedSignature = numericValue != 0;
 
             s_allowDetachedSignature = allowDetachedSignature;
```

Comment:
This clearly only returns false.  But should then be deleted and the "what if it returned true?" codepath deleted.